docs/binder.rst

```diff
@@ -0,0 +1,61 @@
+Using PRIMO on Binder
+=====================
+
+The :doc:`Quickstart <quickstart>` tutorial walks through the 5 easy steps for installing and running PRIMO on any
+machine. However, we recognize that not every user has access to the required IT permissions on their machines to
+carry out the installation. For such users, we have made PRIMO available via a free and public service through Binder.
+This allows anyone to access, experiment with and run PRIMO from a web browser with **no installation** needed.
+The steps to access PRIMO via Binder follow. Please note that:
+
+* The Binder environment is **public and therefore insecure**. Sensitive data should NOT be uploaded/used on this environment. This environment should be used only for **testing, exploration and experimentation** purposes.
+* The Binder environment is automatically deleted after 10 minutes of inactivity. To avoid lost work, please download your work locally on a periodic basis. 
+  
+Please reach out to the NETL team (primo@netl.doe.gov) for any questions or concerns. 
+We remain committed to helping states implement MERP in any way we can.
```

Comment:
should be implement **"the"** MERP

Reply:
Thank you for the attention to detail. Appreciate it :-)